test/dart_api/value/color_test.dart

```diff
@@ -2,6 +2,7 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+@Skip("TODO(nweiz): Update these for the new Color API")
```

Comment:
Should this have a tracking bug?

Reply:
Oops! I've just updated these in this PR, PTAL